packages/layers/base/app/composables/usePartyTable.ts

```diff
@@ -0,0 +1,143 @@
+import { isEqual } from 'es-toolkit'
+import { h } from 'vue'
+import type { BadgeProps } from '@nuxt/ui'
+import type { ExpandedState } from '@tanstack/vue-table'
+import {
+  TableColumnName,
+  TableColumnMailingAddress,
+  TableColumnDeliveryAddress
+} from '#components'
+
+export const usePartyTable = <T>() => {
```

Comment:
Same sort of thing here - thoughts on this being our base business table composable? I'm thinking this is pretty close to being usable for any sort of editable data table we'll have for the business forms. It could define all the different columns and we could extrapolate those into utils (similar to how we do schemas) so it stays clean?
Maybe something to leave for later, but curious what you think